src/components/screen/ChangePw/index.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  CloseButton,
+  Header,
+  InnerContainer,
+  StyledTextInput,
+  Title,
+} from './styles';
+import React, {
+  ReactElement,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import Button from '../../shared/Button';
+import { DefaultNavigationProps } from '../../../types';
+import { Ionicons } from '@expo/vector-icons';
+import { SafeAreaView } from 'react-navigation';
+import { getString } from '../../../../STRINGS';
+import { useThemeContext } from '@dooboo-ui/native-theme';
+
+export interface Props {
+  navigation: DefaultNavigationProps;
+}
+
+function ChangePw(props: Props): ReactElement {
+  const { navigation } = props;
+  const { theme } = useThemeContext();
+  const [isValidCurrentPw, setCurrentPwValid] = useState(false);
+  const [currentPw, setCurrentPw] = useState('');
+  const [newPw, setNewPw] = useState('');
+  const [validationWord, setValidationWord] = useState('');
+  const close = (): void => {
+    navigation.goBack();
+  };
+  const validateCurrent = (): void => {
+    try {
+      if (currentPw === 'right') {
+        setCurrentPwValid(true);
+      } else {
+        throw Error(getString('PASSWORD_MUST_MATCH'));
+      }
+    } catch (err) {
+      Alert.alert('', err.message);
+    }
+  };
+  const changePassword = async (): Promise<void> => {
+    if (newPw === validationWord) {
+      // TODO change password api call
+      Keyboard.dismiss();
+      Alert.alert('', 'Password changed.', [
+        {
+          text: getString('OK'),
+          onPress: (): void => {
+            close();
+          },
+        },
+      ]);
+    } else {
+      Alert.alert(getString('PASSWORD_MUST_MATCH'));
+    }
+  };
+  const [keyboardOffset, setKeyboardOffset] = useState(0);
+  const onKeyboardShow = (event: KeyboardEvent): void => setKeyboardOffset(event.endCoordinates.height);
+  const onKeyboardHide = (): void => setKeyboardOffset(0);
+  const keyboardDidShowListener = useRef<EmitterSubscription>();
+  const keyboardDidHideListener = useRef<EmitterSubscription>();
+
+  useEffect(() => {
+    keyboardDidShowListener.current = Keyboard.addListener('keyboardWillShow', onKeyboardShow);
+    keyboardDidHideListener.current = Keyboard.addListener('keyboardWillHide', onKeyboardHide);
+
+    return (): void => {
+      keyboardDidShowListener.current && keyboardDidShowListener.current.remove();
+      keyboardDidHideListener.current && keyboardDidHideListener.current.remove();
+    };
+  }, []);
+  return (
+    <SafeAreaView style={{ flex: 1, backgroundColor: theme.background, paddingBottom: keyboardOffset }}>
+      <Header>
+        <Title>{getString('PASSWORD_CHANGE')}</Title>
+        <CloseButton testID="closeBtn" onPress={close}>
+          <View
+            style={{
+              padding: 8,
+            }}
+          >
+            <Ionicons name="md-close" size={24} color={theme.fontColor} />
+          </View>
+        </CloseButton>
+      </Header>
```

Comment:
I think removing this part will prevent such side effects on the header. Try using header which is already implemented in `react-navigation` that will handle such cases.